packages-next/fields/src/types/timestamp/views/index.tsx

```diff
@@ -11,50 +10,106 @@ import {
   FieldControllerConfig,
   FieldProps,
 } from '@keystone-next/types';
-import { jsx } from '@keystone-ui/core';
-import { FieldContainer, FieldLabel, TextInput } from '@keystone-ui/fields';
+import { jsx, Inline, Stack } from '@keystone-ui/core';
+import { FieldContainer, FieldLabel, TextInput, DatePicker, DateType } from '@keystone-ui/fields';
+import { TextInputProps } from '@keystone-ui/fields/src/TextInput';
 
 // TODO: Bring across the datetime/datetimeUtc interfaces, date picker, etc.
-
-function formatOutput(value: string) {
-  if (!value) return '';
-  const date = new Date(value);
-  return `${date.toLocaleDateString()} ${date.toLocaleTimeString()}`;
+interface TimePickerProps extends TextInputProps {
+  format: '12hr' | '24hr'
 }
 
+const TimePicker = ({ autoFocus, onBlur, disabled, onChange, format = '24hr', value }: TimePickerProps) => {
+  return (
+    <TextInput
+      autoFocus={autoFocus}
+      // maxLength={format === '24hr' ? 5 : 7}
+      disabled={disabled}
+      onChange={onChange}
+      onBlur={onBlur}
+      placeholder={ format === '24hr' ? '00:00' : '00:00am' }
+      value={value}
+    />
+  );
+};
+
+
+
 export const Field = ({
   field,
   value,
   onChange,
   forceValidation,
-  autoFocus,
 }: FieldProps<typeof controller>) => {
-  const [touched, setTouched] = useState(false);
+  const [touchedFirstInput, setTouchedFirstInput] = useState(false);
+  const [touchedSecondInput, setTouchedSecondInput] = useState(false);
+  const showValidation = touchedFirstInput && touchedSecondInput || forceValidation;
+
+  const showDateError = (dateValue: DateType) => {
```

Comment:
@JedWatson  what's our stance on validation and Date Picker/ Time Picker as design-system components? I'm leaning towards making our DS components as dumb as possible and letting users handle validation if they need it.

We can provide validation helpers from the component if necessary, but hesitant to make inferences about validity at a DS level.